pkg/runner/deb/packageconfig.go

```diff
@@ -31,5 +34,48 @@ func (p *PackageConfig) UpdatePackageGroup(g *PackageGroup) error {
 		g.Add(*p.Override...)
 	}
 
+	if p.OverrideDir != nil {
+		overrides := make([]Package, 0, len(g.packages))
+		for _, pkg := range g.packages {
+			if pkg.LocalPath != nil {
+				continue
+			}
+			if localDeb, err := findLocalDeb(pkg, p.OverrideDir); err == nil {
+				overrides = append(overrides, Package{
+					Name:      pkg.Name,
+					LocalPath: &localDeb,
+				})
+			}
+		}
+
+		if len(overrides) > 0 {
+			g.Add(overrides...)
+		}
+	}
+
 	return nil
 }
+
+// Function to support 'override' directory.
+// Ignore version numbers in packages.
+var ErrNoOverrrideDir = errors.New("no Override Directory given")
+var ErrMultiplePackages = errors.New("multiple packages found")
+var ErrPackageNotFound = errors.New("package not found")
+
+func findLocalDeb(pkg Package, dir *string) (string, error) {
+	var patt = pkg.Name + "_*_*.deb"
+
+	matches, err := filepath.Glob(filepath.Join(*dir, patt))
+	if err != nil {
+		return "", err
+	}
+	if len(matches) == 0 {
```

Comment:
This shouldn't be an error right?  It just means there's no package in the override directory.